client/containers/NavContainer.js

```diff
@@ -12,6 +12,9 @@ function mapDispatchToProps (dispatch) {
   return {
     setLevel: (levelNum) => {
       dispatch(actions.setLevel(levelNum))
+    },
+    toggleSound: () => {
+      dispatch(actions.toggleSound())
```

Comment:
You should use `actions.createAction(actions.TOGGLE_SOUND)` for simple actions that have no payload.